src/user-event/press/__tests__/longPress.real-timers.test.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Pressable, Text } from 'react-native';
+import { render, screen } from '../../../pure';
+import { userEvent } from '../..';
+import * as WarnAboutRealTimers from '../utils/warnAboutRealTimers';
+
+const mockWarnAboutRealTimers = jest
+  .spyOn(WarnAboutRealTimers, 'warnAboutRealTimers')
+  .mockImplementation();
+
+describe('userEvent.longPress with real timers', () => {
+  beforeEach(() => {
+    jest.useRealTimers();
+  });
+
+  test('calls onLongPress if the delayLongPress is the default one', async () => {
+    const mockOnLongPress = jest.fn();
+    const user = userEvent.setup();
+
+    render(
+      <Pressable onLongPress={mockOnLongPress}>
+        <Text>press me</Text>
+      </Pressable>
+    );
+    await user.longPress(screen.getByText('press me'));
+
+    expect(mockOnLongPress).toHaveBeenCalled();
```

Comment:
Let's add check for `mockWarnAboutRealTimers` in this test instead of having a separate one just for this warning.

Reply:
I prefer having a separate test because the test name makes it very clear what we're testing. Also there is no connection between longPress being called and the warning being emitted so either could fail independently so I think they're better tested in isolation